tokenmagic/module/settings.js

```diff
@@ -198,7 +198,17 @@ export class TokenMagicSettings extends FormApplication {
     }
 
     static configureAutoFPS(enabled = false) {
-        if (enabled) canvas.app.ticker.maxFPS = 0;
+        if (enabled) {
+            Object.defineProperty(MouseInteractionManager.prototype, "_dragThrottleMS", {
+                get: function () {
+                    return this._dragThrottleMS_;
+                },
+                set: function (value) {
+                    this._dragThrottleMS_ = Math.clamped(value !== Infinity ? value : 0, 17, 1000);
+                }
+            });
+            canvas.app.ticker.maxFPS = 0;
+        }
```

Comment:
We need to add a third argument: `{configurable: true}` to the `defineProperty`. Otherwise we get an error if we toggle auto fps off and on again: `TypeError: Cannot redefine property: _dragThrottleMS`.

Reply:
Oh! Shame on me! 😫

Reply:
Done.